packages/cactus-plugin-cc-tx-visualization/src/test/typescript/integration/cctxviz-generate-use-case-dummy-baseline-events.test.ts

```diff
@@ -0,0 +1,241 @@
+import test, { Test } from "tape-promise/tape";
+import { LogLevelDesc, LoggerProvider } from "@hyperledger/cactus-common";
+import { pruneDockerAllIfGithubAction } from "@hyperledger/cactus-test-tooling";
+import { IPluginCcTxVisualizationOptions } from "../../../main/typescript";
+import { CcTxVisualization } from "../../../main/typescript/plugin-cc-tx-visualization";
+import { CrossChainModelType } from "../../../main/typescript/models/crosschain-model";
+import { randomUUID } from "crypto";
+import {
+  FabricContractInvocationType,
+  RunTxReqWithTxId,
+} from "@hyperledger/cactus-plugin-ledger-connector-fabric";
```

Comment:
Where is a connector being initialized? we need to test with real connectors, not mocks. Is the observable functionality enabled in the respective connector?